.github/scripts/hermetic_library_generation.sh

```diff
@@ -94,10 +91,12 @@ docker run \
   -v "$(pwd):${workspace_name}" \
   -v "$HOME"/.m2:/home/.m2 \
   -e GENERATOR_VERSION="${generator_version}" \
-  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}"
+  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}" \
+  --baseline-generation-config-path="${workspace_name}/${baseline_generation_config}" \
```

Comment:
I thought the generation process is still going to be skipped inside the scripts if there is no diff between the two generation configs?